### PR TITLE
Tools: Add React Storefront framework to the developer tools.

### PR DESCRIPTION
**Pull request by ianand**

### CODE REVIEW

**Comment by ianand** [comment]
I've signed the CLA.

**Comment by pbakaus** [comment]
@googlebot rescan

**Comment by pbakaus** [comment]
@googlebot I've manually confirmed that the commits in this PR have a signed CLA, and that the person who committed the PRs is the same as the PR opener (not sure what triggered this error).

File: pages/shared/data/tools.yaml [hunk 1]
```diff
@@ -275,6 +275,17 @@ developer:
       - email
       - ads
       - stories
+  - name: React Storefront
+    official: true
```
Comment (pbakaus) [inline pages/shared/data/tools.yaml]: Could you please remove the `official` attribute here? That one is reserved for tools built by the AMP team. Thanks!

Comment (ianand) [reply pages/shared/data/tools.yaml]: @pbakaus ah my bad. didn't realize that. i've pushed a correction.

**Pull request merged**